src/main/menu/electron-menu-items.test.ts

```diff
@@ -0,0 +1,132 @@
+/**
```

Comment:
Please put test files under a `__test__` folder

Reply:
Why? There are tests already in codebase that are not in `__tests__` directory.

Reply:
I'd argue for having tests in a tests folder (being a bit vague to not bikeshed too much). If Lens codebase is not consistent wrt, we should strive to make it so. I think file navigation becomes less convenient if test files are located next to application code.

Reply:
You might be both right:

Sometimes unit tests specify real life behaviour instead of workings of certain technical entrypoint (i.e. a file). For this, `__tests__`makes sense, as the entrypoint becomes less relevant. This does not happen in current codebase yet. Some call this integration testing or functional testing, but let's not get into semantics.

At other times unit tests specify workings of a technical detail. For this, placing the test file next to entrypoint of unit makes sense, as the presence of tests (or lack thereof) is a valuable piece of information not to be swept away in a directory. When doing this, inconvenience in navigation is oftentimes sign of another problem, namely files not being organised in directories properly. Tests should not make directories feel cluttered.